apps_product_creator/models/product_template.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    odoo_module_id = fields.Many2one(
+        'odoo.module',
+        'Odoo Module',
+        readonly=True,
+    )
+    technical_name = fields.Char(
+        "Technical Name",
+        related="odoo_module_id.technical_name",
+        store=True,
+    )
+    image_module = fields.Binary(
+        'Icon Image',
+        readonly=True,
+        related="odoo_module_id.image",
+        store=True,
```

Comment:
Here the image_module is a Binary and also a related field, why is it set as stored?
Since it is a binary, I would expect nobody will need to search/sort this field... In my opinion there's not need to store it.